### ISSUE TITLE
Recommendations for noise reduction and echo cancellation

### ISSUE THREAD

**Issue by c469591** [BearWare/TeamTalk5#1828]
### Description

Hi, I have some suggestions about teamtalk, I will organize the questions as follows→

1. I think the mobile version can separate noise cancellation and echo cancellation into two separate switches.
Sometimes I just want to activate the echo cancellation and don't want to have the additional noise cancellation to further reduce the sound quality. 1.
The phone board proposes to detect whether the headphones are plugged in, turn off the noise cancellation function when the headphones are plugged in, and automatically turn on noise cancellation and echo cancellation when the headphones are not connected.
This is because if I turn on the noise cancellation and echo cancellation, what I hear when I listen to the poor headphones are all mono to, whether the channel is stereo or not, what I hear is all mono to
And if I don't turn on noise cancellation and echo cancellation, after I unplug the headset, the other party's voice will be played through the phone's speaker, and then radioed by my phone's microphone, causing the other party to hear echoes.
1. I hope the computer version can add the quick key setting to turn on the noise cancellation and echo function

Thank you!

### Application

- [ ] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by CoBC** [comment]
2. Very difficult to find shortcuts that are not used, it was already a 
big task just to change shortcuts for increase/decrease sound volume 
when we did it.

**Comment by amirmahdifard** [comment]
yeah exactly, you can just simply go to the settings and do it from there, then developers can have time to work on other good features to add
